cluster/v1alpha1/types.go

```diff
@@ -0,0 +1,40 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope="Namespaced"
+
+// ManagedClusterSetBinding projects a managed cluster set into a certain namespace.
```

Comment:
Good start at the description, I do think that we should enrich this with information about how users can expect to use these resources. Do they act the same as ManagedClusters? What do they allow you to do? I know we're still sorting these properties out, but let's record the key ones here (in subsequent PRs) as we do.